<launch>
  <node pkg="nodelet" type="nodelet" name="depth_image_proc_nodelet_manager" args="manager" output="screen" />

  <node pkg="nodelet" type="nodelet" name="depth_to_cloud" args="load depth_image_proc/point_cloud_xyz depth_image_proc_nodelet_manager" output="screen">
    <remap from="image_rect" to="/xtion/depth_registered/image_raw"/>
    <remap from="camera_info" to="/xtion/depth_registered/camera_info"/>
    <remap from="points" to="/xtion/depth_pointsros"/>
  </node>
</launch>
